### PR TITLE
Astro 2/7 release notes

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

File: astro/release-notes.md [hunk 1]
```diff
@@ -23,6 +23,17 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+## February 7, 2023
+
+### Additional improvements 
+
+- The page for new Astro users who are not yet part of an Organization has been updated to provide more clear first steps.
+
+### Bug fixes
+
+- Removed broken date filtering functionality from the Lineage UI.
+- Fixed an issue where a you could unintentionally turn off DAG deploys on a Deployment by triggering an image deploy from older versions of the Astro CLI.
```
Comment (jwitz) [inline astro/release-notes.md:35]:
```suggestion
- Fixed an issue where a you could unintentionally turn off DAG deploys on a Deployment by triggering an image deploy from an older version of the Astro CLI.
```

File: astro/release-notes.md [hunk 2]
```diff
@@ -23,6 +23,17 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+## February 7, 2023
+
+### Additional improvements 
+
+- The page for new Astro users who are not yet part of an Organization has been updated to provide more clear first steps.
+
+### Bug fixes
+
+- Removed broken date filtering functionality from the Lineage UI.
+- Fixed an issue where a you could unintentionally turn off DAG deploys on a Deployment by triggering an image deploy from an older version of the Astro CLI.
```
Comment (kwcanuck) [inline astro/release-notes.md:35]:
```suggestion
- Fixed an issue where triggering an image deploy from an older version of the Astro CLI could unintentionally turn off DAG deploys on a Deployment.
```
If you don't accept this edit, you'll need to change _an issue where a you_ to _an issue where you_. Remove the _a_.

File: astro/release-notes.md [hunk 3]
```diff
@@ -23,6 +23,17 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
 
 **Latest CLI Version**: 1.10.0 ([Release notes](cli/release-notes.md))
 
+## February 7, 2023
+
+### Additional improvements 
+
+- The page for new Astro users who are not yet part of an Organization has been updated to provide more clear first steps.
```
Comment (kwcanuck) [inline astro/release-notes.md:30]:
```suggestion
- The instructions on the <page name> for new Astro users who are not yet part of an Organization have been improved to make joining a Workspace easier.
```
Does this page have a name? For example, the log in page? What is the intent of the page? I've made some guesses, but I recommend a rewrite to include more information.

**Review by kwcanuck — APPROVED** [review]

File: astro/release-notes.md (hunk 3 above)
Comment (jwitz) [reply astro/release-notes.md:30]: Unfortunately no, but I think "welcome page" is a good replacement.

File: astro/release-notes.md [hunk 4]
```diff
@@ -23,6 +23,17 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
 
+## February 7, 2023
+
+### Additional improvements 
+
+- The instructions on the welcome page for new Astro users who are not yet part of an Organization have been improved to make getting started easier.
+
+### Bug fixes
+
+- Removed broken date filtering functionality from the Lineage UI.
```
Comment (jwitz) [inline astro/release-notes.md:34]:
```suggestion
- Removed nonfunctioning date filtering functionality from the Lineage UI.
```

File: astro/release-notes.md [hunk 5]
```diff
@@ -23,6 +23,17 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
 
+## February 7, 2023
+
+### Additional improvements 
+
+- The page for new Astro users who are not yet part of an Organization has been updated to provide more clear first steps.
+
+### Bug fixes
+
+- Removed broken date filtering functionality from the Lineage UI.
```
[the remark this answers is not in the document]
Comment (amccabe-astronomer) [reply astro/release-notes.md]: It's really a nit, I think "Removed date filtering functionality" sidesteps it :)

You can feel free to leave as-is, just a suggestion.

**Review by amccabe-astronomer — APPROVED** [review]
👍 for my part.

**Pull request merged**